src/resources/postcss/tribe-events-full.pcss

```diff
@@ -1422,8 +1422,17 @@ select.tribe-events-dropdown {
 			}
 		}
 	}
+
+	body.admin-bar & {
```

Comment:
is the `&` character necessary here? to my (limited) knowledge, the ampersand can be used to _precede_ selectors, but not follow them like it is here...right?